report-viewer/src/utils/ComparisonUtils.ts

```diff
@@ -7,3 +7,7 @@
 export function generateLineCodeLink(index: number, fileName: string, line: number): string {
   return String(index).concat(fileName).concat(String(line))
 }
+
+export function toTwoDecimals(num: number) {
+  return num.toFixed(2)
```

Comment:
I would argue that this function only makes understanding harder, not easier

Reply:
Would you suggest always trimming numbers in place where they are displayed?

Reply:
Yes, that's probably easier